python/mxnet/io.py

```diff
@@ -635,6 +633,9 @@ class NDArrayIter(DataIter):
         How to handle the last batch. This parameter can be 'pad', 'discard' or
         'roll_over'. 'roll_over' is intended for training and can cause problems
         if used for prediction.
+        If 'pad', the last batch will be padded with data starting from the begining
+        If 'discard', the last batch will be discarded
+        If 'roll_over', the remaining elements will be rolled over to the next iteration
```

Comment:
'roll_over' option is described in line 634. Can this be merged with that rather than describe 'roll_over' twice.